price-feeder/config/config.go

```diff
@@ -97,37 +97,52 @@ type (
 	// Telemetry defines the configuration options for application telemetry.
 	Telemetry struct {
 		// Prefixed with keys to separate services
-		ServiceName string `toml:"service_name" validate:"required"`
+		ServiceName string `toml:"service_name"`
 
 		// Enabled enables the application telemetry functionality. When enabled,
 		// an in-memory sink is also enabled by default. Operators may also enabled
 		// other sinks such as Prometheus.
-		Enabled bool `toml:"enabled" validate:"required"`
+		Enabled bool `toml:"enabled"`
```

Comment:
Should we have this required? WDYT?

Reply:
"required" for booleans means that it must be true, since they default to false if not set

so, that would produce `Error: Key: 'Config.Telemetry.Enabled' Error:Field validation for 'Enabled' failed on the 'required' tag` if required was set here

Reply:
Gotcha, that seems a bit odd. I'd expect that it means it must be supplied, but not necessarily true.

Feel free to merge 👍

Reply:
Yeah I thought so too, but apparently the "required" tag just will check to make sure that a variable is not set to its default value 🤷 ref: https://github.com/go-playground/validator/issues/319